packages/react-native-renderer/src/ReactFiberConfigFabric.js

```diff
@@ -543,19 +542,14 @@ export function finalizeContainerChildren(
   container: Container,
   newChildren: ChildSet,
 ): void {
-  if (!enableFabricCompleteRootInCommitPhase) {
-    completeRoot(container.containerTag, newChildren);
-  }
+  // Noop - children will be replaced in replaceContainerChildren
 }
 
 export function replaceContainerChildren(
   container: Container,
   newChildren: ChildSet,
 ): void {
-  // Noop - children will be replaced in finalizeContainerChildren
-  if (enableFabricCompleteRootInCommitPhase) {
-    completeRoot(container.containerTag, newChildren);
-  }
+  completeRoot(container.containerTag, newChildren);
```

Comment:
This should probably be renamed to align with React terminology. Really these should exist in both forms. Both completeRoot (which would really do layout eagerly) and commitRoot. This is the commitRoot part.